python/res/job_queue/job_queue_manager.py

```diff
@@ -58,80 +55,62 @@ def __init__(self, queue):
         self.queue = queue
         super(JobQueueManager, self).__init__(c_ptr)
 
-    def status_timestamp(self):
-        """
-        Will return a datetime object corresponding to the last status change
-        """
-        ts = self._status_timestamp()
-        return ts.datetime()
-
-    def progress_timestamp(self, job_index = None):
-        """Will return the timestamp of last progress update.
-
-        By default the timestamp will be global, i.e. for all jobs, but if you
-        pass a job_index as optional argument the timestamp will correspond to
-        that job.
-        """
-        if job_index is None:
-            ts = self._global_progress_timestamp()
-        else:
-            ts = self._iget_progress_timestamp(job_index)
-
-        return ts.datetime()
-
-
     def get_job_queue(self):
         return self.queue
 
     def stop_queue(self):
-        self._stop_queue( )
+        self.get_job_queue().kill_all_jobs()
 
     def startQueue(self , total_size , verbose = False ):
         self._start_queue( total_size , verbose )
 
     def getNumRunning(self):
-        return self._get_num_running(  )
+        return self.get_job_queue().count_status(JobStatusType.JOB_QUEUE_RUNNING)
 
     def getNumWaiting(self):
-        return self._get_num_waiting( )
+        return self.get_job_queue().count_status(JobStatusType.JOB_QUEUE_WAITING)
 
     def getNumPending(self):
-        return self._get_num_pending( )
+        return self.get_job_queue().count_status(JobStatusType.JOB_QUEUE_PENDING)
 
 
     def getNumSuccess(self):
-        return self._get_num_success( )
+        return self.get_job_queue().count_status(JobStatusType.JOB_QUEUE_DONE)
+
 
     def getNumFailed(self):
-        return self._get_num_failed(  )
+        return self.get_job_queue().count_status(JobStatusType.JOB_QUEUE_FAILED)
+
 
     def isRunning(self):
-        return self._is_running( )
+        return self.get_job_queue().is_running()
 
     def free(self):
         self._free( )
 
+    #resolve ambiguity done vs success
```

Comment:
Please turn this comment into a descriptive issue

Reply:
Will do ...